utils/problems.py

```diff
@@ -415,6 +451,34 @@ async def fetch_problems_solved_and_rank(
         submit_stats_global = matched_user["submitStatsGlobal"]
         ac_submission_num = submit_stats_global["acSubmissionNum"]
 
+        languages_problem_count = matched_user["languageProblemCount"]
+
+        tags_problem_count = matched_user["tagProblemCounts"]
+        advanced_tags_count = tags_problem_count["advanced"]
+        intermediate_tags_count = tags_problem_count["intermediate"]
+        fundamental_tags_count = tags_problem_count["fundamental"]
```

Comment:
Defining each of these variables is unecessary. Just remove these, and just refer to the actually value (aka tags_problem_count["fundamental"], wherever necessary)